packages/mux-player-react/src/lazy.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useEffect, useState } from 'react';
+
+import ConditionalSuspense from './ConditionalSuspense';
+import useIsBrowser from './useIsBrowser';
+import useIsIntersecting from './useIsIntersecting';
+
+import type { MuxPlayerProps, MuxPlayerRefAttributes } from './index';
+
+const MuxPlayerIndex = React.lazy(() => import('./index'));
+
+interface FallbackProps extends MuxPlayerProps {
+  onIntersection?: () => void;
+}
+const Fallback = (props: FallbackProps) => {
+  const { style, className, onIntersection, placeholder } = props;
+
+  const intersectionRef = React.useRef<HTMLElement>(null);
+  const isIntersecting = useIsIntersecting(intersectionRef);
+
+  useEffect(() => {
+    if (isIntersecting && onIntersection) {
+      onIntersection();
+    }
+  }, [isIntersecting, onIntersection]);
+
+  return (
+    <>
+      {/* 
+        Why do we have a mux-player element before the mux-player bundle is even loaded?
+        Before the bundle is loaded, this mux-player element just acts like a div. 
+        However, by calling this placeholder "mux-player", 
+        it now gets the same CSS applied to it that the eventual "real" mux-player element will.
+      */}
+      {/* TODO: can we add mux-player to JSX.IntrinsicElements */}
+      {/* @ts-ignore */}
+      <mux-player
+        ref={intersectionRef}
+        data-mux-player-react-lazy-placeholder
+        className={className || ''}
+        style={{
+          // default mux-player styling
+          aspectRatio: '16/9',
+          display: 'block',
+          backgroundColor: 'var(--media-control-background, #000)',
```

Comment:
Is this the correct way to make sure the placeholder background is the same color as the player background?

Reply:
I'm not sure if this is the right CSS var. `--media-control-background` changes the background of each control separately.

better one I think is `var(--media-background-color, #000)` which is the background color of `<media-controller>`

https://github.com/muxinc/media-chrome/blob/5ea0d24bc18de58587b8f90137f72cfde585879f/src/js/media-container.js#L38-L38

Reply:
Thank you!